packages/react-core/src/index.ts

```diff
@@ -44,6 +44,8 @@ export {
   useTimeout,
   useDataState,
   DataState,
+  useDropZone,
```

Comment:
We're doing this to prevent a direct dependency on the react package

Reply:
Correct